### PR TITLE
feat: updates and enhancements

### PR DESCRIPTION
**Pull request by nasbench**

### CODE REVIEW

File: rules/windows/file/file_event/file_event_win_lolbin_gather_network_info_script_output.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      type: similar
+status: experimental
+description: Detects creation of files which are the results of executing the built-in reconnaissance script "C:\Windows\System32\gatherNetworkInfo.vbs".
+references:
+    - https://posts.slayerlabs.com/living-off-the-land/#gathernetworkinfovbs
+    - https://www.mandiant.com/resources/blog/trojanized-windows-installers-ukrainian-government
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/02/08
+tags:
+    - attack.persistence
```
Comment (frack113) [inline rules/windows/file/file_event/file_event_win_lolbin_gather_network_info_script_output.yml:16]: more Discovery  for me

File: rules/windows/registry/registry_set/registry_set_office_outlook_enable_unsafe_client_mail_rules.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+title: Outlook Security EnableUnsafeClientMailRules Value Was Enabled
```
Comment (frack113) [inline rules/windows/registry/registry_set/registry_set_office_outlook_enable_unsafe_client_mail_rules.yml:1]:
```suggestion
title: Outlook Security EnableUnsafeClientMailRules Setting Enabled
```
`Was Enabled` gives me the impression of something long in the past

File: rules/windows/registry/registry_set/registry_set_persistence_outlook_homepage.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-title: Persistent Outlook Landing Pages
+title: Potential Persistence Via Outlook Home Page
```
Comment (frack113) [inline rules/windows/registry/registry_set/registry_set_persistence_outlook_homepage.yml:1]: modified too

**Review by frack113 — APPROVED** [review]

File: rules/windows/process_creation/proc_creation_win_lolbin_cscript_gathernetworkinfo.yml [hunk 4]
```diff
@@ -1,28 +0,0 @@
-title: GatherNetworkInfo.vbs Script Usage
-id: 575dce0c-8139-4e30-9295-1ee75969f7fe
-status: test
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_lolbin_cscript_gathernetworkinfo.yml:3]: obligatory `git mv` reminder :innocent:

File: rules/windows/process_creation/proc_creation_win_malware_blackbyte_ransomware.yml [hunk 5]
```diff
@@ -1,29 +1,26 @@
-title: BlackByte Ransomware Patterns
+title: Potential BlackByte Ransomware Activity
 id: 999e8307-a775-4d5f-addc-4855632335be
 status: test
-description: This command line patterns found in BlackByte Ransomware operations
+description: Detects command line patterns used by BlackByte Ransomware in different operations
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_malware_blackbyte_ransomware.yml:4]:
```suggestion
description: Detects command line patterns used by BlackByte ransomware in different operations
```

File: rules/windows/process_creation/proc_creation_win_reg_defender_exclusion.yml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 title: Suspicious Windows Defender Folder Exclusion Added Via Reg.EXE
 id: 48917adc-a28e-4f5d-b729-11e75da8941f
 status: experimental
-description: Qbot used reg.exe to add Defender folder exceptions for folders within AppData and ProgramData.
+description: Detects usage of "reg.exe" to add Defender folder exclusions. Qbot has been seen using this technique to add exlcusions for folders within AppData and ProgramData.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_defender_exclusion.yml:4]:
```suggestion
description: Detects the usage of "reg.exe" to add Defender folder exclusions. Qbot has been seen using this technique to add exlcusions for folders within AppData and ProgramData.
```

File: rules/windows/registry/registry_set/registry_set_office_outlook_enable_load_macro_provider_on_boot.yml [hunk 7]
```diff
@@ -0,0 +1,28 @@
+title: Potential Persistence Via Outlook LoadMacroProviderOnBoot Setting
+id: 396ae3eb-4174-4b9b-880e-dc0364d78a19
+status: experimental
+description: Detects the modification of Outlook Setting "LoadMacroProviderOnBoot" which if enabled allow the automatic loading of any configured VBA project/module
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_office_outlook_enable_load_macro_provider_on_boot.yml:4]:
```suggestion
description: Detects the modification of Outlook setting "LoadMacroProviderOnBoot" which if enabled allows the automatic loading of any configured VBA project/module
```

File: rules/windows/registry/registry_set/registry_set_office_outlook_enable_macro_execution.yml [hunk 8]
```diff
@@ -1,12 +1,13 @@
-title: Outlook C2 Registry Key
+title: Outlook Macro Execution Without Warning Setting Enabled
 id: e3b50fa5-3c3f-444e-937b-0a99d33731cd
 status: test
-description: Detects the modification of Outlook Security Setting to allow unprompted execution. Goes with win_outlook_c2_macro_creation.yml and is particularly interesting if both events occur near to each other.
+description: Detects the modification of Outlook Security Setting to allow unprompted execution of macros.
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_office_outlook_enable_macro_execution.yml:4]:
```suggestion
description: Detects the modification of Outlook security setting to allow unprompted execution of macros.
```

File: rules/windows/registry/registry_set/registry_set_office_outlook_enable_unsafe_client_mail_rules.yml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+date: 2023/02/08
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    category: registry_set
+    product: windows
+detection:
+    selection:
+        EventType: Setvalue
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_office_outlook_enable_unsafe_client_mail_rules.yml:21]:
```suggestion
        EventType: SetValue
```

File: rules/windows/registry/registry_set/registry_set_office_outlook_security_settings.yml [hunk 10]
```diff
@@ -1,13 +1,16 @@
-title: Change Outlook Security Setting in Registry
+title: Outlook Security Settings Updated - Registry
 id: c3cefdf4-6703-4e1c-bad8-bf422fc5015a
+related:
+    - id: a166f74e-bf44-409d-b9ba-ea4b2dd8b3cd
+      type: similar
 status: test
-description: Change outlook email security settings
+description: Detect changes to the registry values related to outlook security settings
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_office_outlook_security_settings.yml:7]:
```suggestion
description: Detects changes to the registry values related to outlook security settings
```

**Review by phantinuss — APPROVED** [review]

File: rules/windows/process_creation/proc_creation_win_lolbin_cscript_gathernetworkinfo.yml (hunk 4 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_lolbin_cscript_gathernetworkinfo.yml:3]: This one something weird happened XD, I did a git mv but then I saved it again mistakenly and I happen to have 2 of the same file with different names 😛 not my fault (git is hard)

**Pull request merged**